catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/content/operation/impl/ReadStorageResponseImpl.java

```diff
@@ -27,6 +27,15 @@
     private ContentItem contentItem;
 
     /**
+     * Instantiates an empty, new ResponseImpl
+     *
+     * @param request    - the original request
```

Comment:
Nit: You don't need to put a `-` between the parameter name and the description. The javadoc tool takes care of that for you.